uprotocol/uattributes.proto

```diff
@@ -111,6 +111,8 @@ enum UMessageType {
     UMESSAGE_TYPE_PUBLISH = 1 [(uprotocol.ce_name) = "pub.v1"];    // Publish or notification
     UMESSAGE_TYPE_REQUEST = 2 [(uprotocol.ce_name) = "req.v1"];    // Request 
     UMESSAGE_TYPE_RESPONSE = 3 [(uprotocol.ce_name) = "res.v1"];   // Response
+    UMESSAGE_TYPE_NOTIFICATION = 4 [(uprotocol.ce_name) = "pub.v1"]; // Notification are identified in 
```

Comment:
shouldn't the option's value be something different from that for PUBLISH?

Reply:
I'm trying to make it backwards compatible with what we've already implemented that has it defined as pub event, otherwise it will break our existing implementations.

Reply:
Probably my ignorance showing. How are the option values being used in implementations? Wouldn't folks use the enum value? I didn't see the option values for message type being used in `up-rust` at least.

Reply:
> How is this supposed to work when using the option value in a transport protocol binding? This would result in the wrong message type being used at the receiving side of a notification, wouldn't it?

The protobuf option value is specifically called "ce_name" and not just "name" for that purpose, to help map to CloudEvents.

Reply:
I am not sure if I understand your point. If I want to send a notification with `UAttributes.type = UMessageType::UMESSAGE_TYPE_NOTIFICATION` then it now seems to depend on how the message type is being mapped to the transport's PDU whether the receiver will see it as a Publish or Notification message. For example if the attributes are being serialized to protobuf and put into a message property, then the receiver will see type NOTIFICATION. If the type's *ce_name* is being put into a message property, then the receiver will see type PUBLISH. Is this intended (and maybe acceptable/not a problem)?

Reply:
> Probably my ignorance showing. How are the option values being used in implementations? Wouldn't folks use the enum value? I didn't see the option values for message type being used in `up-rust` at least.

Not yet. However, we use the option values of other attributes already in up-rust.

Reply:
@sophokles73 based on the feedback from todays meeting, I've added notification type of cloudevent message.